packages/web-components/fast-components/src/tooltip/tooltip.stories.ts

```diff
@@ -25,12 +26,30 @@ function onAnchorMouseEnter(e: MouseEvent): void {
     tooltipInstance.anchorElement = e.target as HTMLElement;
 }
 
+//check changing anchor by setting attribute
+function onAnchorMouseEnterAttribute(e: MouseEvent): void {
```

Comment:
This duplicative function can be combined with the original by looking at the id of `e.target`

Reply:
It's a sample, not package code, so I prefer the more "obvious" example.